src/components/Feed/Feed.js

```diff
@@ -39,17 +35,12 @@ const Feed = ({ profileIsVisible, searchResults }) => {
 
   const displayPostsInFeed = (posts) => {
```

Comment:
I don't know where this would go, I'm guessing in the Feed? But a future feature is definitely adding some pagination so we can show like 10-20 posts at a time.

Reply:
Definitely!